actors/user/src/main/java/org/sunbird/user/dao/impl/UserOrgDaoImpl.java

```diff
@@ -0,0 +1,39 @@
+package org.sunbird.user.dao.impl;
+
+import java.util.Map;
+
+import org.sunbird.cassandra.CassandraOperation;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.helper.ServiceFactory;
+import org.sunbird.learner.util.Util;
+import org.sunbird.models.user.org.UserOrg;
+import org.sunbird.user.dao.UserOrgDao;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public final class UserOrgDaoImpl implements UserOrgDao {
+
+  private final Util.DbInfo userOrgDbInfo = Util.dbInfoMap.get(JsonKey.USER_ORG_DB);
```

Comment:
Suggestion is to move away from maintaining a static list of db info map in memory. The table name can be defined locally as a constant in this class itself.